src/typeahead/test/typeahead.spec.js

```diff
@@ -1049,15 +1071,15 @@ describe('typeahead deprecation', function() {
     expect($log.warn.calls.argsFor(1)).toEqual(['typeahead is now deprecated. Use uib-typeahead instead.']);
     expect($log.warn.calls.argsFor(2)).toEqual(['typeahead-popup is now deprecated. Use uib-typeahead-popup instead.']);
   }));
-  
+
```

Comment:
please watch your formatting and only touch lines relevant to your code changes.